test/commands/user/mailshot/send_test.rb

```diff
@@ -16,12 +16,42 @@ class User::Mailshot::SendTest < ActiveSupport::TestCase
     end
   end
 
-  test "not sent if receive_product_updates is false" do
+  test "community_launch: email not sent if receive_product_updates is false" do
     user = create :user
     user.communication_preferences.update(receive_product_updates: false)
 
     assert_no_enqueued_jobs do
       User::Mailshot::Send.(user, :community_launch)
     end
   end
+
+  test "company_support_donor: sends email" do
```

Comment:
I've not added a test for being able to opt out, as I think this will be sent regardless (nobody will have had to option to opt-out as that setting is introduced in this PR)